manifests/m/Microsoft/DSC/Preview/3.1.1/Microsoft.DSC.Preview.installer.yaml

```
@@ -0,0 +1,59 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Microsoft.DSC.Preview
+PackageVersion: 3.1.1
```

```suggestion
PackageVersion: 3.1.1.0
```

I _believe_ validation fails if the PackageVersion isnt an **exact** match of the MSIX version, though the latest pipeline run would confirm that